NEWS.md

```diff
@@ -6,6 +6,19 @@
 
 ## NEW FEATURES
 
+1. New function `fcase(.,default)` has been implemented in C, [#3823](https://github.com/Rdatatable/data.table/issues/3823). It is comparable to `dplyr::case_when`. Please see `?data.table::fcase` for more details.
```

Comment:
I would point to SQL first, SQL `CASE WHEN` is the logical ancestor of `dplyr` and `data.table` versions I think. Especially since we don't use the formula interface.